data/monster/dreamhaunters/shock head.xml

```diff
@@ -49,7 +49,7 @@
 		<immunity invisible="1"/>
 	</immunities>
 	<voices interval="5000" chance="10">
-		<voice sentence="<brrzz>"/>
```

Comment:
Issue with unescape XML while in parsing files to search errors duplicated items or wrong information in monsters.